flopy/mf6/utils/postprocessing.py

```diff
@@ -60,7 +60,7 @@ def get_structured_faceflows(
     shape = (grb.nlay, grb.nrow, grb.ncol)
     frf = np.zeros(shape, dtype=float).flatten()
     fff = np.zeros(shape, dtype=float).flatten()
-    flf = np.zeros(shape, dtype=float)
+    flf = np.zeros(shape, dtype=float).flatten()
```

Comment:
Can you add a test to show that this is necessary?

Reply:
[test.zip](https://github.com/modflowpy/flopy/files/7900992/test.zip)

Please run the two small testing programs, called 1)test_w_modfified_version.py and 2)test_w_nonmodfified_version.py, to see the difference in the output, thanks.